### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by bhuvy2** [illinois-cs241/illinois-cs241.github.io#86]
Currently, our site is in a kinda state for accessibility. There should be a jekyll plugin that checks for accessibility on the final web page and sends a custom email if any violations were found.

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML
